database/src/main/java/info/metadude/android/eventfahrplan/database/extensions/CursorExtensions.kt

```diff
@@ -40,6 +40,17 @@ inline fun Cursor.getLong(columnName: String): Long = getLong(getColumnIndexOrTh
  */
 inline fun Cursor.getString(columnName: String): String = getString(getColumnIndexOrThrow(columnName))
 
+/**
+ * Returns the value of the requested column as a string or null.
+ *
+ * The result and whether this method throws an exception when the column value is null or the
+ * column type is not a string type is implementation-defined.
+ *
+ * @see Cursor.getColumnIndexOrThrow
+ * @see Cursor.getString
+ */
+inline fun Cursor.getStringOrNull(columnName: String): String? = getString(getColumnIndexOrThrow(columnName))
```

Comment:
```suggestion
/**
 * Returns the value of the requested column as a string or null.
 *
 * The result and whether this method throws an exception when the column type is not a string type
 * is implementation-defined.
 *
 * @see Cursor.getColumnIndexOrThrow
 * @see Cursor.getStringOrNull
 */
inline fun Cursor.getStringOrNull(columnName: String): String? = getStringOrNull(getColumnIndexOrThrow(columnName))
```

with `import androidx.core.database.getStringOrNull`

Reply:
Much nicer. Thanks. Changed in 0714fdd0771a25652917a74919f1ab435a3da221.